easybuild/easyconfigs/m/MathGL/MathGL-2.4.1-foss-2018a.eb

```diff
@@ -0,0 +1,61 @@
+easyblock = 'CMakeMake'
+
+name = 'MathGL'
+version = '2.4.1'
+
+homepage = 'http://mathgl.sourceforge.net/doc_en/Main.html'
+description = """MathGL is ...
+ a library for making high-quality scientific graphics under Linux and Windows;
+ a library for the fast data plotting and data processing of large data arrays;
+ a library for working in window and console modes and for easy embedding into other programs;
+ a library with large and growing set of graphics. """
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+toolchainopts = {'openmp': True}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+patches = ['%(name)s-%(version)s_FL.patch']
+checksums = [
+    '751488bb9454c057d6e2828867dd56e00507c84a47efa1b8546da55a3455a588',  # mathgl-2.4.1.tar.gz
+    '8061cd86e4a5509e174deff111906bf19a8b1c6c4317543e65cac382e3b600f5',  # MathGL-2.4.1_FL.patch
+]
+
+builddependencies = [
+    ('CMake', '3.10.2'),
+    ('gettext', '0.19.8.1', '-libxml2-2.9.7'),
+    ('pkg-config', '0.29.2'),
+    ('intltool', '0.51.0', '-Perl-5.26.1'),
+]
+
+dependencies = [
+    ('FLTK', '1.3.4'),
+    ('libjpeg-turbo', '1.5.3'),
+    ('libpng', '1.6.34'),
+    ('GSL', '2.4'),
+    ('Mesa', '17.3.6'),
+    ('freeglut', '3.0.0'),
+    ('HDF5', '1.10.1'),
+    ('zlib', '1.2.11'),
+    ('Qt5', '5.10.1'),
+]
+
+configopts = '-Denable-double=ON '
+configopts += '-Denable-opengl=ON '
+configopts += '-Denable-openmp=ON '
+configopts += '-Denable-zlib=ON '
+configopts += '-Denable-png=ON '
+configopts += '-Denable-jpeg=ON '
+configopts += '-Denable-gsl=ON '
+configopts += '-Denable-hdf5=ON '
+configopts += '-Denable-glut=ON '
+configopts += '-Denable-fltk=ON '
+configopts += '-Denable-qt5=ON '
```

Comment:
I'd prefer collapsing these on to a line or two.

Or, if you insist on having one per line, at least order them alphabetically :)